pset-functions.js

```diff
@@ -148,6 +242,22 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+function checkDoubleLetter(word) {
+   let result = false
+   if (word.length === 1) {
+      result = false
+   } else {
+      for (i=1; i<word.length; i++) { // should prob use for while loop
+         if (word.charAt(i) === word.charAt(i-1)) {
+            result = true
+         }
+      }   
+   }
+   return result
+}
+
+console.log(checkDoubleLetter("rune"))
```

Comment:
Lance, you are crushing this homework. You are keeping edge cases in mind and make great use of running code inside of your if/else statements. Similar to question 2 above, on line 251, you can replace the condition inside the if statement from (word.charAt(i) === word.charAt(i-1)) to  (word[i] === word[i - 1]). Nothing major but might help other read through the code a little easier.